controllers/task.js

```diff
@@ -838,11 +858,12 @@ const rateUserTask = async (req, res, next) => {
 	}
 
 	let task = await getTaskById(data);
-	console.log(task.data)
 	if (task.error || !task.data) {
```

Comment:
for error case message should be 500

Reply:
No sir it would be 400 only because we have used 400 for not found errors in whole rateUserTask controller

Reply:
but you are handling error case for status 400 for not found it is correct but for error in query it should be 500